curvefs/src/client/s3/client_s3_cache_manager.cpp

```diff
@@ -522,7 +535,9 @@ int FileCacheManager::ReadFromS3(const std::vector<S3ReadRequest> &requests,
             DataCachePtr dataCache = std::make_shared<DataCache>(
             s3ClientAdaptor_, chunkCacheManager.get(), chunkPos,
             (*responses)[i].GetBufLen(), (*responses)[i].GetDataBuf());
-            chunkCacheManager->AddReadDataCache(dataCache);
+            if (!curvefs::client::common::FLAGS_enableCto) {
+                chunkCacheManager->AddReadDataCache(dataCache);
```

Comment:
why?

Reply:
When cto isconsistency  enabled, the write cache conversion to read cache is forcibly disabled.